docs/getting_started/get_started.md

```diff
@@ -35,34 +35,47 @@ Otherwise, to login, you must get your user credentials from the administrator a
 
 ![Login screen](img/login_form.png "Login screen")
 
-### Edit user profile
+### View and edit user profile
```

Comment:
I'm tempted to go further, and have two "View user profile" and "Edit user profile" sections

Reply:
I don't think it's necessary to go so granular.

Reply:
I prefer having a noun here, to show that we're task oriented

Reply:
I'm agree. And for consistency of the ToC

- Access the Back Office
    - View and edit user profile
    - Change the password and other user settings